client/my-sites/themes/themes-selection.jsx

```diff
@@ -107,6 +108,26 @@ const ThemesSelection = React.createClass( {
 		this.props.incrementPage();
 	},
 
+	//intercept preview and add primary and secondary
+	getOptions( theme ) {
+		const options = this.props.getOptions( theme );
+		const wrappedPreviewAction = ( action ) => {
+			return ( themeObj ) => {
+				const defaultAction = ( theme.price && options.purchase ) ? options.purchase : this.props.defaultOption;
```

Comment:
The const should be called `defaultOption` to distinguish theme options from actual (Redux) actions.